src/SUMMARY.md

```diff
@@ -87,6 +87,8 @@
     - [Behavior considered undefined](behavior-considered-undefined.md)
     - [Behavior not considered unsafe](behavior-not-considered-unsafe.md)
 
+- [Beneath `std`](beneath-std.md)
```

Comment:
I don't like this name. At all. It implies this is information only useful for `std`, since it's stuff beneath it. And it also implies the things that `std` can do that other crates cannot. Perhaps something like "Runtime", since the things you have to implement are all going to be what's effectively the Rust Runtime. Then we would also be able to document what the standard behaviour does.